paegan-transport/meta.yaml

```diff
@@ -0,0 +1,41 @@
+package:
+    name: paegan-transport
+    version: "1.0.3dev"
+
+source:
+  git_url: https://github.com/axiom-data-science/paegan-transport.git
+  git_tag: db8f957f5f379def417f93c1941d3f20dbbf16bb
```

Comment:
Loved it!  Avoid master as much as you can.  (Releases as better though :stuck_out_tongue_winking_eye:)